subgraph/src/fleek-nfa.ts

```diff
@@ -370,13 +452,13 @@ export function handleChangeAccessPointCreationStatus(event: ChangeAccessPointCr
 
   if (accessPointEntity) {
     if (status == 0) {
```

Comment:
A swtich-case cames handy for not using a lot of else if 😆

Reply:
that's what I was searching for lol! In rust we had `match` and that was the keyword I used in my searches so I couldn't find this.